test/grakn_test.exs

```diff
@@ -2,61 +2,147 @@ defmodule GraknTest do
-        Grakn.query!(conn, Grakn.Query.graql("define age sub attribute, datatype long;"))
-        Grakn.query!(conn, Grakn.Query.graql("define person sub entity, has name, has age;"))
-      end, type: Grakn.Transaction.Type.write())
-      {:ok, _} = Grakn.transaction(context[:conn], fn conn ->
-        Grakn.query(conn, Grakn.Query.graql("insert $x isa person;"))
-      end, type: Grakn.Transaction.Type.write())
-      {:ok, _} = Grakn.transaction(context[:conn], fn conn ->
-        Grakn.query!(conn, Grakn.Query.graql("match $x isa person; insert $x has name \"test\";"))
-        Grakn.query!(conn, Grakn.Query.graql("match $x isa person; insert $x has age 100;"))
-      end, type: Grakn.Transaction.Type.write())
+      {:ok, _} =
+        Grakn.transaction(
+          context[:conn],
+          fn conn ->
+            Grakn.query!(conn, Grakn.Query.graql("define name sub attribute, datatype string;"))
+            Grakn.query!(conn, Grakn.Query.graql("define age sub attribute, datatype long;"))
+            Grakn.query!(conn, Grakn.Query.graql("define person sub entity, has name, has age;"))
+          end,
+          keyspace: @keyspace,
+          type: Grakn.Transaction.Type.write()
+        )
+
+      {:ok, _} =
+        Grakn.transaction(
+          context[:conn],
+          fn conn ->
+            Grakn.query(conn, Grakn.Query.graql("insert $x isa person;"))
+          end,
+          keyspace: @keyspace,
+          type: Grakn.Transaction.Type.write()
+        )
+
+      {:ok, _} =
+        Grakn.transaction(
+          context[:conn],
+          fn conn ->
+            Grakn.query!(
+              conn,
+              Grakn.Query.graql("match $x isa person; insert $x has name \"test\";")
+            )
+
+            Grakn.query!(conn, Grakn.Query.graql("match $x isa person; insert $x has age 100;"))
+          end,
+          keyspace: @keyspace,
+          type: Grakn.Transaction.Type.write()
+        )
     end
 
     test "incorrect schema", context do
-      {:ok, _} = Grakn.transaction(context[:conn], fn conn ->
-        Grakn.query!(conn, Grakn.Query.graql("define name sub attribute, datatype string;"))
-        Grakn.query!(conn, Grakn.Query.graql("define city sub entity, has name;"))
-      end, type: Grakn.Transaction.Type.write())
-      {:error, _} = Grakn.transaction(context[:conn], fn conn ->
-        {:error, reason} = Grakn.query(conn, Grakn.Query.graql("insert $x isa city; $x has city_name \"wrong\";"))
-        Grakn.rollback(conn, reason)
-      end, type: Grakn.Transaction.Type.write())
+      {:ok, _} =
+        Grakn.transaction(
+          context[:conn],
+          fn conn ->
+            Grakn.query!(conn, Grakn.Query.graql("define name sub attribute, datatype string;"))
+            Grakn.query!(conn, Grakn.Query.graql("define city sub entity, has name;"))
+          end,
+          keyspace: @keyspace,
+          type: Grakn.Transaction.Type.write()
+        )
+
+      {:error, _} =
```

Comment:
`assert`. Sorry for repetition, let's use `assert`(s) also in the rest of the file